### PR TITLE
refactor(saveDialog): Simplify remote electron require

### PR DESCRIPTION
**Pull request by lgeiger**
Remove unnecessary `require('electron')`

### CODE REVIEW

File: test/setup.js [hunk 1]
```diff
@@ -101,15 +101,9 @@ mock('electron', {
...
-        return {
-          'dialog': {
-            'showSaveDialog': function(config) { },
-          }
-        };
-      }
-    },
+    'dialog': {
+      'showSaveDialog': function(config) { },
+    }
```
Comment (rgbkrk) [inline test/setup.js]: Missing a comma right here which causes tests to fail.

**Review by rgbkrk — CHANGES_REQUESTED** [review]

File: test/setup.js (hunk 1 above)
Comment (lgeiger) [reply test/setup.js]: Ooops. Actually the mock is unnecessary right now but it could be helpful for future tests.

**Review by rgbkrk — APPROVED** [review]

File: test/setup.js (hunk 1 above)
Comment (lgeiger) [reply test/setup.js]: I should've added a `caution-late-night-PR` label 😂

Comment (rgbkrk) [reply test/setup.js]: Heh.

**Comment by rgbkrk** [comment]
Are there any more like this?

**Comment by lgeiger** [comment]
> Are there any more like this?

I don't know. Discovered this at random.

**Pull request merged**